### DISCUSSION TITLE
editare file home assistant docker compose

### DISCUSSION THREAD

**Discussion by dibummer** [indomus/forum#2967] [HUB: Home Assistant]
ciao,
ho installato home assistant docker compose su base Debian;
sino a pochi giorni fà utilizzavo home assistant HASSIO e quindi disponevo degli ADDON;
 al momento ho difficolta a reperire un metodo per edittare i file yaml di configuration;
gentilmente qualcuno potrebbe indicarmi una soluzione.
grazie

**Comment by snakuzzo** [comment]
Non c'è nessun metodo specifico. Se avevi un addon "tizio", cerchi la documentazione su internet su come tirare su un container che ti permetta di fare le stesse cose (o quasi).

**Reply by dibummer** [reply]
ok. grazie

**Reply by Scialla** [reply]
Cerca Config Editor su HACS

**Reply by dibummer** [reply]
l'ho installato e configurato, ma di fatto non riesco anzi non sono capace  ad utilizzarlo per la configurazione del file configuration.yaml.
ho installato samba su debian e vedo tutte le cartelle e file di home assistant; poi con visual code riesco a vedere tutto il contenuto ma non posso modificare i file e/o cartelle, se potresti indicarmi/aiutarmi come edittare i file. quest'ultima procedura la preferisco. grazie

**Reply by Scialla** [reply]
non riesci a modificarli perché il container di Homeassistant lavora come root. 
Modifica i permessi ai file di configurazione così poi riesci a editarli e a salvare le modifiche

**Reply by dibummer** [reply]
scusami, come dovrei fare a cambiare i file di configurazione? mi  servirebbe un piccolo input. Grazie

**Reply by snakuzzo** [reply]
il piccolo input te l'ha già dato. Devi modificare i permessi dei file

**Reply by dibummer** [reply]
la difficoltà da dove quale comando? 
chmod [ugoa] [+-=] [rwx] nomefile dopo essermi posizionato nella cartella home assistant?
oppure a tutta la cartella home assistant con
sudo chown -R ( nome utente) /home/deb/homeassistant/

**Reply by snakuzzo** [reply]
conosci `chown` e `chmod` ? ... dall'avatar che usi dovresti saperlo :)

**Reply by Scialla** [reply]
fallo ricursivo sulla cartella

**Reply by dibummer** [reply]
Grazie scialla
